oak/server/oak_node.cc

```diff
@@ -138,8 +140,17 @@ static wabt::Result ReadModule(const std::string module_bytes, wabt::interp::Env
   return result;
 }
 
+const std::string Sha256Hash(const std::string& data) {
+  std::vector<uint8_t> digest(SHA256_DIGEST_LENGTH);
```

Comment:
Pick either "digest" or "hash" for consistency?

Reply:
Done.